OneSchedule.Domain.Models/Strategies/IStrategy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace OneSchedule.Domain.Models.Strategies
+{
+    public interface IStrategy
+    {
+        Task Execute(EventDomain eventDomain);
```

Comment:
If the method is public and async use Async suffix in method's name, e.g. ExecuteAsync